src/uglify/versions.js

```diff
@@ -1,5 +1,4 @@
 export default {
-  uglify: require('uglify-es/package.json').version, // eslint-disable-line global-require
+  uglify: require('@fabiosantoscode/uglify-es/package.json').version, // eslint-disable-line global-require
   plugin: require('../../package.json').version, // eslint-disable-line global-require
 };
-
```

Comment:
Here :smi

Reply:
mom, I didn't intentionally touch it (maybe `eslint --fix` on `precommit`)

Reply:
Very strange, let's return this newline